### DISCUSSION TITLE
How to provide a caption for an image in the body section of the markup?

### DISCUSSION THREAD

**Discussion by jarmos-san** [adityatelange/hugo-PaperMod#241] [Q&A] [answered]
While the cover image can be captioned pretty well & which is a great feature. So, thanks for including it. But I realised, there's no way to caption an image in the body of the article. Or perhaps, I missed out some information regarding it.

Anyway, I look forward to finding a solution for it.

**Comment by adityatelange** [comment, marked as answer]
checkout hugo's `figure` shortcode

**Comment by jarmos-san** [comment]
For anyone else stumbling across this thread. Refer to the [Shortcodes > Figure](https://gohugo.io/content-management/shortcodes/#figure) section of the Hugo docs.

**Reply by aronreisx** [reply]
Thanks jarmos!
